components-part1/src/components/Logo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLogo = styled.div`
+  color: #f65261;
+  height: 24px;
+  width: 151px;
+  margin-left: 20px;
+  transform: translateY(-10px);
+  & .logo_1 {
+    font-family: Montserrat;
```

Comment:
we can simply share some of the properties not changed in `.logo_1` and `.logo_21.

Reply:
fixed, have them share same properties